src/app/components/profile-indicator/profile-indicator.component.ts

```diff
@@ -1,17 +1,34 @@
 import { Component, OnInit } from '@angular/core';
 import { faUserCircle } from '@fortawesome/free-solid-svg-icons';
+import { User } from 'src/app/models/user';
+import { ActivatedRoute } from '@angular/router';
 
 @Component({
   selector: 'app-profile-indicator',
   templateUrl: './profile-indicator.component.html',
   styleUrls: ['./profile-indicator.component.scss']
 })
 export class ProfileIndicatorComponent implements OnInit {
-  faUserCircle = faUserCircle;
+  user: User;
+  faUserCircle: any;
 
-  constructor() { }
+  constructor(private router: ActivatedRoute) {
+    this.faUserCircle = faUserCircle;
+    this.user = {
+      displayName: '',
+      email: '',
+      photoURL: ''
+    };
+  }
 
   ngOnInit() {
+    this.router.params.subscribe(
+      parameters => {
```

Comment:
this does not need to be sent through the url

i just tested this 

  getUser() {
    return this.firebaseAuth.user;
  }

Reply:
Cool will make use of this.firebaseAuth.user